src/app/components/navbar/navbar.component.html

```diff
@@ -4,7 +4,9 @@
         <img src="assets/images/logo.svg" alt="">
       </a>
       <div class="navbar-nav">
-        <a class="d-none d-sm-block nav-link text-uppercase fw-bold" routerLink="/login">Login</a>
+        <a *ngIf="router.url !== '/login/logged-in'" class="d-none d-sm-block nav-link text-uppercase fw-bold" routerLinkActive="border-bottom border-primary border-3 active" routerLink="/login">Login</a>
+        <a *ngIf="router.url === '/login/logged-in'" class="d-none d-sm-block nav-link text-uppercase fw-bold" href="/">My account</a>
+        <a *ngIf="router.url === '/login/logged-in'" class="d-none d-sm-block nav-link text-uppercase fw-bold" href="/">Settings</a>
```

Comment:
Don't forget to replace in future, may be add TODO

Reply:
done